### DISCUSSION TITLE
Integrated window display

### DISCUSSION THREAD

**Discussion by allblue** [1Remote/1Remote#479] [Ideas]
**Is your feature request related to a problem? Please describe.**

It is possible to integrate the address book and the remote connection into one window, such as the left side is a multi-level address tree, and the right main area is a remote connection tab. This is also the display mode of mainstream remote tools, such as mRemoteNG, mobaxterm, RemoteDesktopManager, etc., splitting into two is a bit strange.

可将地址簿和远程连接整合为一个窗口，如左侧为多级地址树，右侧主区域为远程连接标签页。这也是主流远程工具的显示模式，如mRemoteNG、mobaxterm、RemoteDesktopManager等，拆分成二个有点怪怪的。

**Describe the solution you'd like**
See the description above.

见上方描述

**Describe alternatives you've considered**
no

**Additional context**
no

**Comment by majkinetor** [comment]
This is not currently planned. What is your exact problem with current design?

**Reply by allblue** [reply]
Switching between seperate  windows is not very convenient, it’s a bit anti-user.

**Reply by majkinetor** [reply]
Switching shouldn't be a problem, there is launcher hotkey ALT + M to launch connection, and we plan to make main window minimize/close when connection is launched within it as another option (along with adding its own hotkey).

Having a tree there is also not very convenient, since then you need to manage it as it occupies space that is better used to host remotes. That was one of my constant problems with mRemoteNG for example.
